paddle/fluid/operators/activation_mkldnn_op.cc

```diff
@@ -100,9 +100,6 @@ void eltwise_forward(const framework::ExecutionContext &ctx,
   const T *x_data = x->data<T>();
   T *y_data = y->mutable_data<T>(ctx.GetPlace());
 
-  PADDLE_ENFORCE(x->dims().size() == 2 || x->dims().size() == 4,
-                 "Input dim must be with 2 or 4");
-
```

Comment:
why remove the `PADDLE_ENFORCE` in this PR?

Reply:
As can not verify dims be 5 and others, so I add the value checking when the dims equal 3.